### ISSUE TITLE
OSError: [WinError 10013] when using jupyter notebook with anaconda3

### ISSUE THREAD

**Issue by QingGo** [jupyter/notebook#1936]
I have intall python3, anaconda2 and anaconda3 in my window8.1 system, but i can only open jupyter notebook with anaconda2. when using jupyter notebook with anaconda3 or python3, it will jump out an "OSError: [WinError 10013 Attempts to access the socket are made in a way that access is not allowed.(translated)"

```
C:\Windows\system32>where anaconda
E:\Users\Zeng\Anaconda2\Scripts\anaconda.exe
E:\Users\Zeng\Anaconda3\Scripts\anaconda.exe

C:\Windows\system32>where jupyter
E:\Users\Zeng\Anaconda2\Scripts\jupyter.exe
E:\Users\Zeng\Anaconda3\Scripts\jupyter.exe
C:\Users\Zeng\AppData\Local\Programs\Python\Python35\Scripts\jupyter.exe

C:\Windows\system32>E:\Users\Zeng\Anaconda2\Scripts\jupyter.exe notebook
[I 16:37:46.539 NotebookApp] [nb_conda_kernels] enabled, 2 kernels found
[I 16:37:48.627 NotebookApp] 鉁?nbpresent HTML export ENABLED
[W 16:37:48.628 NotebookApp] 鉁?nbpresent PDF export DISABLED: No module named n
bbrowserpdf.exporters.pdf
[I 16:37:48.644 NotebookApp] [nb_conda] enabled
[I 16:37:49.030 NotebookApp] [nb_anacondacloud] enabled
[I 16:37:49.696 NotebookApp] Serving notebooks from local directory: C:\Windows\
system32
[I 16:37:49.697 NotebookApp] 0 active kernels
[I 16:37:49.697 NotebookApp] The Jupyter Notebook is running at: http://localhos
t:38888/
[I 16:37:49.698 NotebookApp] Use Control-C to stop this server and shut down all
 kernels (twice to skip confirmation).
[I 16:38:19.703 NotebookApp] Interrupted...
[I 16:38:19.703 NotebookApp] Shutting down kernels

C:\Windows\system32>E:\Users\Zeng\Anaconda3\Scripts\jupyter.exe notebook
[I 16:38:34.727 NotebookApp] [nb_conda_kernels] enabled, 2 kernels found
Traceback (most recent call last):
  File "E:\Users\Zeng\Anaconda3\lib\site-packages\tornado\platform\common.py", l
ine 44, in __init__
    self.writer.connect(connect_address)
OSError: [WinError 10013] 以一种访问权限不允许的方式做了一个访问套接字的尝试。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Users\Zeng\Anaconda3\Scripts\jupyter-notebook-script.py", line 5, in
<module>
    sys.exit(notebook.notebookapp.main())
  File "E:\Users\Zeng\Anaconda3\lib\site-packages\jupyter_core\application.py",
line 267, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "E:\Users\Zeng\Anaconda3\lib\site-packages\traitlets\config\application.p
y", line 652, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-7>", line 2, in initialize
  File "E:\Users\Zeng\Anaconda3\lib\site-packages\traitlets\config\application.p
y", line 87, in catch_config_error
    return method(app, *args, **kwargs)
  File "E:\Users\Zeng\Anaconda3\lib\site-packages\notebook\notebookapp.py", line
 1071, in initialize
    self.init_webapp()
  File "E:\Users\Zeng\Anaconda3\lib\site-packages\notebook\notebookapp.py", line
 919, in init_webapp
    self.http_server.listen(port, self.ip)
  File "E:\Users\Zeng\Anaconda3\lib\site-packages\tornado\tcpserver.py", line 12
7, in listen
    self.add_sockets(sockets)
  File "E:\Users\Zeng\Anaconda3\lib\site-packages\tornado\tcpserver.py", line 13
9, in add_sockets
    self.io_loop = IOLoop.current()
  File "E:\Users\Zeng\Anaconda3\lib\site-packages\tornado\ioloop.py", line 213,
in current
    return IOLoop.instance()
  File "E:\Users\Zeng\Anaconda3\lib\site-packages\tornado\ioloop.py", line 161,
in instance
    IOLoop._instance = IOLoop()
  File "E:\Users\Zeng\Anaconda3\lib\site-packages\tornado\util.py", line 278, in
 __new__
    instance.initialize(*args, **init_kwargs)
  File "E:\Users\Zeng\Anaconda3\lib\site-packages\zmq\eventloop\ioloop.py", line
 133, in initialize
    super(ZMQIOLoop, self).initialize(impl=impl, **kwargs)
  File "E:\Users\Zeng\Anaconda3\lib\site-packages\tornado\ioloop.py", line 705,
in initialize
    self._waker = Waker()
  File "E:\Users\Zeng\Anaconda3\lib\site-packages\tornado\platform\common.py", l
ine 48, in __init__
    detail[0] != errno.WSAEADDRINUSE):
TypeError: 'OSError' object is not subscriptable
```

I have tried to close the windows firewall but the result is same and have not found the port 8888 being occupied with command "netstat -na". I have no idea what is wrong. Anyone experienced this problem before?

**Comment by takluyver** [comment]
This is not related to the socket that we open on port 8888, but a 'waker' socket which tornado itself creates. It looks like there is at least one bug in tornado (the `TypeError: 'OSError' object is not subscriptable` error), though I'm not sure whether that would fix the problem you see.

Can you try reporting that traceback on https://github.com/tornadoweb/tornado/issues ? Feel free to ping me on the issue when you do.

**Comment by QingGo** [comment]
Thank for your answer @takluyver 
I have not used tornado before so I readlly don't know what the traceback mean . I will report that traceback on /tornado/issues. Hope someone know what is wrong.

**Comment by takluyver** [comment]
The traceback is the bottom 2/3 of what you posted above: it shows what code was running when the error happened.

**Comment by QingGo** [comment]
I found the reason! I install COMODO mouths ago and  shortly after that uninstall it. However, I never thought it would still work for my computer. Even I close my windows defender and firework, the uninstalled COMODO still prevent my computer for some specific connection. The solution way is: 
1. Open the Network Connections folder and view available connections.
2. Right-click the connection that you want to configure, and then click Properties.
3. on the General tab, under This connection uses the following items, uninstall the item include "comodo".
All the problems above were solved!
